tests/integration/routes/test_feedback.py

```diff
@@ -62,20 +62,101 @@ def test_valid_feedback(self):
+        self.assertInBody(
+            "You have reached the maximum number of times for submitting feedback"
+        )
+
+    def test_feedback_error_message_on_post_when_limit_reached(self):
+        # Given I launch and complete the test_feedback questionnaire, and provide
+        # feedback the maximum number of times
+        self._launch_and_complete_questionnaire()
+
+        for _ in range(0, EQ_FEEDBACK_LIMIT):
+            self.get(self.SEND_FEEDBACK_URL)
+            self.post(
+                {
+                    "feedback-type": "Page design and structure",
+                    "feedback-text": "Feedback",
+                }
+            )
+
+        self.get(self.SEND_FEEDBACK_URL)
+
+        # When I post more feedback
+        self.post(
+            {
+                "feedback-type": "Page design and structure",
+                "feedback-text": "Feedback",
+            }
+        )
+
+        # Then I should see an appropriate error
+        self.assertInBody(
+            "You have reached the maximum number of times for submitting feedback"
+        )
+
+    def test_send_feedback_back_breadcrumb(self):
+        # Given I launch and complete the test_feedback questionnaire
+        self._launch_and_complete_questionnaire()
+
+        # When I view the send feedback page
+        self.get(self.SEND_FEEDBACK_URL)
+
+        # Then I should see the Back breadcrumb
+        self.assertInSelectorCSS("Back", class_="breadcrumb__link")
+
+    def test_submission(self):
+        # Given I launch and complete the test_feedback questionnaire, and provide
+        # feedback
         self._launch_and_complete_questionnaire()
         self.get(self.SEND_FEEDBACK_URL)
         self.post(
             {"feedback-type": "Page design and structure", "feedback-text": "Feedback"}
         )
         self.assertInUrl(self.SENT_FEEDBACK_URL)
 
-        # When I go to the feedback send page again
+        # When I request the send feedback page
         self.get(self.SEND_FEEDBACK_URL)
 
-        # Then I get redirected to the sent page
-        self.assertInUrl(self.SENT_FEEDBACK_URL)
+        # Then I go to the feedback send page
+        self.assertInUrl(self.SEND_FEEDBACK_URL)
+
+    def test_multiple_submissions(self):
+        # Given I launch and complete the test_feedback questionnaire, and provide
+        # multiple feedback submissions
+        self._launch_and_complete_questionnaire()
+        self.get(self.SEND_FEEDBACK_URL)
+        self.post(
+            {"feedback-type": "Page design and structure", "feedback-text": "Feedback"}
+        )
+        self.get(self.SEND_FEEDBACK_URL)
+        self.post(
+            {"feedback-type": "Page design and structure", "feedback-text": "Feedback"}
+        )
+
+        # When I request the send feedback page
+        self.get(self.SEND_FEEDBACK_URL)
+
+        # Then I go to the feedback send page
+        self.assertInUrl(self.SEND_FEEDBACK_URL)
```

Comment:
I'm not sure why we need to go back to the send page here. We should just assert that we're on the feedback sent page after the second `post`.